src/resources/base.js

```diff
@@ -143,6 +143,10 @@ export default api => (
           res = await api[method](url);
         }
 
+        if (path === 'smartrate') {
```

Comment:
Why here instead of in the `getSmartrates` function?

Reply:
The function doesn't yet have the response available to grab specific keys from. It's at this point that we can strip away the key before sending it to the user.